lib/help.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
+const chalk = require('chalk');
+
+const DESCRIPTION_PAD = 20;
+
+/**
+ * Pads a description string to align with a specified offset string.
+ *
+ * @param {string} description - The description to pad.
+ * @param {string} offsetString - The offset string that the description should align with.
+ * @returns {string} - The padded description.
+ */
+function padLeft(description, offsetString) {
+  // Calculate the necessary padding based on the offsetString length
+  const padding = ' '.repeat(Math.max(0, DESCRIPTION_PAD - offsetString.length));
+  return `${padding}${description}`;
+}
+
+/**
+ * Displays a help message for available commands, including descriptions, parameters, and options.
+ *
+ * @param {Object} commands - An object containing information about available commands.
+ */
+function helpCommand(commands) {
+  console.log(chalk.yellow.bold('Paragon Help'));
+  console.log();
+  console.log('Available commands:');
+  console.log();
+
+  Object.keys(commands).forEach(command => {
```

Comment:
I think you can simplify this code a little bit with some destructuring magic, e.g.
```js
Object.entries(commands).forEach(([command, { parameters, description, options }]) => {
```
this way you will able to remove `commands[command].` part that is repeated a bunch of times in the code.

Reply:
Yes, it is good idea!